src/helm/proxy/clients/google_client.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Dict
+
+from helm.common.cache import Cache, CacheConfig
+from helm.common.request import Request, RequestResult, Sequence, Token
+from helm.common.tokenization_request import (
+    TokenizationRequest,
+    TokenizationRequestResult,
+    DecodeRequest,
+    DecodeRequestResult,
+)
+from .client import Client, truncate_sequence
+
+
+class GoogleClient(Client):
+    """
+    Client for the Google models. There isn't an API for their language models.
+    We receive and process completions offline.
+    """
+
+    @staticmethod
+    def convert_to_raw_request(request: Request) -> Dict:
+        return {
```

Comment:
Question: Why don't you use dacite here?

Reply:
Does dacite support `"stop": request.stop_sequences or None`